relion/protocols/protocol_subtract.py

```diff
@@ -86,14 +122,26 @@ def _defineParams(self, form):
                            "That is: *the mask should INCLUDE the part of the "
                            "volume that you wish to KEEP.*")
 
+        form.addParam('invertMask', BooleanParam, default=False,
```

Comment:
not accepted. Use a separate protocol to get your mask right.

Reply:
In the first version of this protocol the required mask was the inverse of the one we ask for in this second version. This option allows users to repeat what they did in the past.  I do not understand why keeping compatibility with the past implementation is wrong but changing the protocol input from one version to the next  is OK.

Reply:
The behaviour you refer to (whether the mask should include the subtracted part or the opposite) was changed in Relion GUI years ago. The "past implementation" is long gone from Scipion plugin as well.

Reply:
I am not convinced but  I have removed the invert option.